### ISSUE TITLE
Investigate Synchronous XMLHttpRequest warning stemming from refactored locations linker

### ISSUE THREAD

**Issue by lorawoodford** [archivesspace/archivesspace#1920]
Discovered while working on #1919 and traced back to changes introduced by #1913 .  By abstracting the locations_holding cell out into its own partial, we are now getting Synchronous XMLHttpRequest warnings in the console.  When time permits, these should be further explored.  The warning specifically stems from the `<%= javascript_include_tag 'locations.location_holdings' %>` on ln 9 of `frontend/app/views/locations/_location_holdings_cell.html.erb`

## Expected Behavior

Previously, no Synchronous XMLHttpRequest warnings were visible in the console, but now the following depreciation warning is given:

![image](https://user-images.githubusercontent.com/15144646/84189122-b77a8b80-aa62-11ea-8adc-a9990e0c49a2.png)

## Steps to Reproduce (for bugs)

1. In a repository with location records, select Manage Top Containers
2. On the locations linker/typeahead line, click the "Browse" option from the downward arrow
3. Note the deprecation warning in the console

Note: This can also be replicated elsewhere in the application where the location linker is provided, such as from within a top container record.

## Your Environment

* Version used: 2.8
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by brianzelip** [comment]
I also found the include_tag that you mention above to be the warning's culprit.

Also, that included js file calls another js file (which I don't really think does anything, so can likely be deleted), all from within a template loop that gets repeated 10x per view, which means 20 js files are imported, which is 18 too many and bad for perf biz.

As far as I can tell, the warning stems from jquery. Instead of looking into that, I came up with 3 ways to possibly move forward.

1. Don't worry about the xhr warning, but solve the duplicated imports, #1938 
2. Solve both problems using inline js in a Rails template, #1937 
3. Solve both problems by listening for "Browse Locations" modal via `setTimeout`, #1936 

I don't find any of the three solutions particularly attractive. But I do find #3 (#1936) the best. Perhaps its arbitrary timeout (400ms) could be increased to allow for an even longer modal DOM load?
